lib/drivers/medtronic600/medtronic600Simulator.js

```diff
@@ -399,7 +399,7 @@ class Medtronic600Simulator {
     return _.filter(this.events, (event) => {
       // filter out zero boluses
       if (event.type === 'bolus') {
-        if (event.normal === 0 && !event.expectedNormal) {
+        if (event.subType === 'normal' && event.normal === 0 && !event.expectedNormal) {
```

Comment:
Was this change approved? I see it as `In Test` [in Jira](https://tidepool.atlassian.net/browse/UPLOAD-8), and don't see any approvals for it.

Reply:
UPLOAD-8 was merged into develop after code review and is now in test as part of release 2.18.0.